### PR TITLE
fix(tooling): cannot run journey tests locally

### PR DESCRIPTION
**Pull request by moriahmaney**

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview for *hopeful-hamilton-4090b9* ready!

Built with commit b4d3dee48c8a804b87a8b36633f902e0dc105f97

https://deploy-preview-785--hopeful-hamilton-4090b9.netlify.com

File: wdio.conf.js [hunk 1]
```diff
@@ -339,6 +326,16 @@ exports.config = {
 };
 
 if (process.env.SAUCE) {
+  const sauceCapabilities = Object.assign({}, chromeCapabilities, {
```
Comment (adamweeks) [inline wdio.conf.js]: We need to handle firefox capabilities as well, correct?

Comment (moriahmaney) [reply wdio.conf.js]: good catch

**Comment by adamweeks** [comment]
Closed via #780

**Pull request closed without merging**